src/docs/patterns/overlay.mdx

```diff
@@ -1,73 +1,190 @@
-    }
-    react={
-        'https://codesandbox.io/s/github/pxblue/react-design-patterns/tree/master?file=/src/pages/overlays/complex-bottom-sheet/index.tsx&initialpath=complex-bottom-sheet'
-    }
-    ionic
-    reactNative
-/>
-
-A bottom sheet is essentially just a container with some attached behaviors. Though it typically contains a simple list, it can be customized to include any content that is required for your application. For example, it can be used for custom sort/filter functionality (seen in the example above), biometric authentication prompts, etc.
+Bottom sheets are content surfaces that slide in from the bottom of the screen, primarily used in mobile applications. There are three types (standard, modal, and expandable), but the one used most often in PX Blue applications is the modal bottom sheet.
+
+<React.Fragment>
+    <ImageGrid
+        regularWidth
+        images={[SimpleBottomsheet]}
+        caption={[
+            '1. On mobile, the modal bottom sheet takes up the full width of the screen.',
+            '2. On desktop, bottom sheets can usually be replaced by a contextual menu instead.',
+        ]}
+    />
+    <DemoCard
+        repository={'bottomsheet'}
+        angular={
+            'https://stackblitz.com/github/pxblue/angular-design-patterns?file=src%2Fapp%2Fpages%2Foverlays%2Fbasic-bottom-sheet%2Fbasic-bottom-sheet.component.ts'
+        }
+        react={
+            'https://codesandbox.io/s/github/pxblue/react-design-patterns/tree/master?file=/src/pages/overlays/basic-bottom-sheet/index.tsx&initialpath=basic-bottom-sheet'
+        }
+        ionic
+        reactNative
+    />
+    <DemoCard
+        repository={'complex-bottomsheet'}
+        angular={
+            'https://stackblitz.com/github/pxblue/angular-design-patterns?file=src%2Fapp%2Fpages%2Foverlays%2Fcomplex-bottom-sheet%2Fcomplex-bottom-sheet.component.ts'
+        }
+        react={
+            'https://codesandbox.io/s/github/pxblue/react-design-patterns/tree/master?file=/src/pages/overlays/complex-bottom-sheet/index.tsx&initialpath=complex-bottom-sheet'
+        }
+        ionic
+        reactNative
+    />
+</React.Fragment>
+
+Modal bottom sheets block interaction with the rest of the application. These typically contain a list of options or actions and are usually activated via an action button in the [app bar](/patterns/appbar).
+
+<React.Fragment>
+    <MaterialDesignDescription
+        title={'Lists'}
+        url={'/patterns/lists'}
+        description={`When using lists in an overlay / bottom sheet, be sure to follow PX Blue's guidelines for lists.`}
+        avatar={<PxblueSmallAltIcon style={{ width: 48, height: 48 }} />}
+    />
+    <Divider />
+</React.Fragment>
+
+# Design Specifications
+
+## Dialog (Mobile)
+
+<ImageGrid regularWidth images={[SpecsMobileDialog1]} />
+<ImageGrid regularWidth images={[SpecsMobileDialog2]} />
```

Comment:
why is this measured from the baseline of the text? Instead of the bottom of the div?

![image](https://user-images.githubusercontent.com/8997218/94299178-48ec0b80-ff35-11ea-91e3-1ebc42c58abd.png)

Reply:
That's just how things lined up...I can make the dialog 2px bigger and go from the edge of the text.